### PR TITLE
docs: meeting minutes 2017-03-23

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/tsc @mikeal

### CODE REVIEW

File: meetings/2017-03-23.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+* James Snell @jasnell (TSC)
+* Rod Vagg @rvagg (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Bryan Hughes @nebrius (TSC)
+* Josh Gavant @joshgav (TSC)
+* William Kapke @williamkapke (observer/NF)
+* Michael Dawson @mhdawson (observer/CTC)
+* Anna Henningsen @addaleax (observer/CTC)
+* Jenn Turner @renrutnnej (observer/Linux Foundation)
```
Comment (bnoordhuis) [inline meetings/2017-03-23.md]: You forgot a certain dashing young programmer...

File: meetings/2017-03-23.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Current charter requires 2/3 to add, 2/3 to remove. Suggestion to reduce to majority for adding to make it easier, while keeping 2/3 for removal.
+
+@bnoordhuis: would prefer to make everything the same as a simple majority.
+
+Any objections?
+
+@jasnell: Have not had to exercise this, don’t anticipate issues.
+
+Conclusion: Go with simple majority/>50% for everything to minimize friction.
```
Comment (bnoordhuis) [inline meetings/2017-03-23.md]: ...and removal after three months of inactivity, not six.

**Comment by jasnell** [comment]
Ping @joshgav ... PTAL at @bnoordhuis' comments

File: meetings/2017-03-23.md (hunk 1 above)
Comment (joshgav) [inline meetings/2017-03-23.md]: Fixed!

File: meetings/2017-03-23.md (hunk 2 above)
Comment (joshgav) [inline meetings/2017-03-23.md]: Updated, PTAL and make sure I captured this correctly. Thanks!

**Comment by joshgav** [comment]
Updated conclusion of membership resignation discussion per Ben's comments above, PTAL to be sure I got it right. Thanks!

**Pull request merged**